tests/www/views/test_views.py

```diff
@@ -297,6 +298,174 @@ def get_task_instance(session, task):
         assert dagrun.get_state() == State.QUEUED
 
 
+@pytest.mark.parametrize(
```

Comment:
Why is this in test_views.py? All the mark functions in views.py do is call `dag.set_task_instance_state` and then redirect. Nothing should be tested in views for this change.

Reply:
There is `test_mark_task_instance_state` which had tests for marking instance state. I thought to add them here since the original issue was about marking it through API. Do you want me to move it to `tests/models/test_dag.py` ?